{{cookiecutter.project_name}}/steps/predict.py

```diff
@@ -11,7 +13,13 @@ def predict_batch(spark_session, model_uri, input_table_name, output_table_name)
     predict = mlflow.pyfunc.spark_udf(
         spark_session, model_uri, result_type="string", env_manager="conda"
     )
-    output_df = table.withColumn("prediction", predict(struct(*table.columns)))
+    output_df = (
+        table.withColumn("prediction", predict(struct(*table.columns)))
+        .withColumn("model_version", lit(model_version))
+        .withColumn("ts", lit(ts))
+        .withColumn("model_version", col("model_version").cast("int"))
+        .withColumn("ts", to_timestamp("ts"))
```

Comment:
Convert them to correct type so that the data monitoring don't need additional step to convert them.